packages/ui/src/proposals/types/ProposalDetails.ts

```diff
@@ -80,6 +80,10 @@ export type UpdateChannelPayoutsDetail = {
   payloadHash?: string
   payloadDataObjectId?: string
 }
+export type UpdatePalletFrozenStatusDetail = {
+  frozen?: boolean
```

Comment:
Here too please change `frozen` to `freeze` because it's the action that the proposal should apply (not the current state).